docs/BreakingChanges/092 Multi-line ASP.Net TextBox spacing changed when using AntiXSSEncoder.md

```diff
@@ -0,0 +1,29 @@
+## 92: Multi-line ASP.Net TextBox spacing changed when using AntiXSSEncoder
```

Comment:
{
  "Id": "92",
  "Title": "Multi-line ASP.Net TextBox spacing changed when using AntiXSSEncoder",
  "Details": "In .NET Framework 4.0, extra lines were inserted between lines of a multi-line text box on postback, if using the `AntiXSSEncoder`. In .NET Framework 4.5, those extra line breaks are not included, but only if the web app is targeting .NET 4.5",
  "Suggestion": "Be aware that 4.0 web apps retargeted to .NET 4.5 may have multi-line text boxes improved to no longer insert extra line breaks. If this is not desirable, the app can have the old behavior when running on .NET Framework 4.5 by targeting the .NET Framework 4.0.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/748052/textbox-with-multiline-textmode-adds-additional-linebreak-to-the-output",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems#id=838859&triage=true&_a=edit",
  "Notes": "Encoder is set in a config file. See bug repro for example.",
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis